src/ReactPikadayComponent.js

```diff
@@ -91,13 +99,20 @@ class ReactPikadayComponent extends React.Component {
     _setupPikaday() {
         const el = this.refs.pikaday;
         const { requestChange } = this._getValueLink(this.props);
-        const { value, onChange, valueLink, ...pikadayOptions } = this.props; // eslint-disable-line no-unused-vars
+        const { value, onChange, valueLink, container, ...pikadayOptions } = this.props; // eslint-disable-line no-unused-vars
+
         const options = Object.assign({}, pikadayOptions, {
             field: el,
+            container: container === true ? this.refs.pikadayContainer :
+                container && container.getDOMNode ? container.getDOMNode() : container,
```

Comment:
it's not possible to get into this case because of `container === true` will always use `this.refs.pikadayContainer`...